katello_certs/hooks/boot/20-certs_update.rb

```diff
@@ -1 +1 @@
-../../../katello/hooks/boot/20-certs_update.rb
\ No newline at end of file
+../../../hooks/boot/20-certs_update.rb
```

Comment:
I always forget these and what they're exactly supposed to do. It would be great to have some integration tests to make sure the functionality actually does what it's supposed to.

Reply:
I'm working on https://github.com/theforeman/forklift/pull/1208 as a first step towards integration testing of installer PRs.

Could we go ahead and merge this, and add tests in a future PR?

Reply:
Yes, tests can come after. Please file a Redmine issue to not lose track of the need.

Reply:
I agree they can come later, but I'm wonder how much we actually verified this continues to work. With the rest of the code I'm decently familiar with how it should work but I'm not that familiar with `foreman-proxy-certs-generate`.